app/src/main/java/com/spoony/spoony/presentation/report/ReportRoute.kt

```diff
@@ -0,0 +1,171 @@
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.LocalLifecycleOwner
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.spoony.spoony.R
+import com.spoony.spoony.core.designsystem.component.button.SpoonyButton
+import com.spoony.spoony.core.designsystem.component.textfield.SpoonyLargeTextField
+import com.spoony.spoony.core.designsystem.component.topappbar.TitleTopAppBar
+import com.spoony.spoony.core.designsystem.theme.SpoonyAndroidTheme
+import com.spoony.spoony.core.designsystem.type.ButtonSize
+import com.spoony.spoony.core.designsystem.type.ButtonStyle
+import com.spoony.spoony.presentation.report.component.ReportRadioButton
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.immutableListOf
+
+@Composable
+fun ReportRoute(
+    viewModel: ReportViewModel = hiltViewModel()
+) {
+    val lifecycleOwner = LocalLifecycleOwner.current
+
+    val state by viewModel.state.collectAsStateWithLifecycle(lifecycleOwner = lifecycleOwner)
+    ReportScreen(
+        options = state.options,
+        selectedOption = state.selectedOption,
+        reportContext = state.reportContext
+    )
+}
+
+@Composable
+private fun ReportScreen(
+    options: ImmutableList<String>,
+    selectedOption: String,
+    reportContext: String
+) {
+    var selectedOption by remember { mutableStateOf(selectedOption) }
+    var reportContext by remember { mutableStateOf(reportContext) }
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .navigationBarsPadding()
+            .statusBarsPadding()
+    ) {
+        TitleTopAppBar(
+            title = "신고하기",
+            onBackButtonClick = {},
+            modifier = Modifier.drawBehind {
+                val strokeWidth = 2.dp.toPx()
+                val y = size.height - strokeWidth / 2
+                drawLine(
+                    color = Color.Gray,
+                    start = Offset(0f, y),
+                    end = Offset(size.width, y),
+                    strokeWidth = strokeWidth
+                )
+            }
+        )
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(top = 64.dp)
+                .background(color = SpoonyAndroidTheme.colors.white)
+                .padding(start = 20.dp, end = 20.dp)
```

Comment:
```suggestion
                .padding(horizontal = 20.dp)
```
P3: 이게 더 낫지 않을까요?????

Reply:
수정했습니다!